TODO

```diff
@@ -0,0 +1,13 @@
+// snek TODO
+
+add optional game modifications
+add more ways to control the game
+add pause
+add easter egg
+add sound effects
+make the field explode on loose
```

Comment:
Maybe, maybe not. I think it's enough if the game "freezes" while in the center is written "Game Over!\nRestart       Quit" and you can choose if you want to restart or quit the game.

Reply:
true, but i'm like EXPLOSIONS xD

i think it would look really cool (probably more EDGY but who cares xD)
but we can make that optional ^^